docs/getting-started-guides/kubeadm.md

```diff
@@ -177,12 +177,43 @@ You must install a pod network add-on so that your pods can communicate with eac
 
 **It is necessary to do this before you try to deploy any applications to your cluster, and before `kube-dns` will start up. Note also that `kubeadm` only supports CNI based networks and therefore kubenet based networks will not work.**
 
-Several projects provide Kubernetes pod networks using CNI, some of which 
-also support [Network Policy](/docs/user-guide/networkpolicies/). See the [add-ons page](/docs/admin/addons/) for a complete list of available network add-ons.
-
-You can install a pod network add-on with the following command: 
-
-    # kubectl apply -f <add-on.yaml>
+Several projects provide Kubernetes pod networks using CNI, some of which also support [Network Policy](/docs/user-guide/networkpolicies/).
+
+You must select a network addon from the drop-down menu below, and correct instructions will appear below.
```

Comment:
Hey @chrismarino, I don't think it matters whether it's used in other docs, as long as it's obvious enough to users. IMHO, it's a very common GUI idiom to have a dropdown select which thing you're looking at. An alternative design could be tabs. But I don't think it makes much of a difference (and dropdown is more scalable than tabs).

I think what really _does_ matter is that [users are having a bad time](https://groups.google.com/forum/#!topic/kubernetes-sig-cluster-lifecycle/FsilgZ_Q2xk):
> So I understand that I need to complete Step 3 first, but I am totally lost on what to do for the network.I feel, this step is quite vague.

Hopefully it's fairly clear how the JS mapping works. Failing that, we have volunteered to maintain the JS.